src/crypto/index.js

```diff
@@ -851,6 +969,135 @@ Crypto.prototype.importRoomKeys = function(keys) {
         },
     );
 };
+
+Crypto.prototype._maybeSendKeyBackup = async function(delay, retry) {
+    if (retry === undefined) retry = true;
+
+    if (!this._sendingBackups) {
+        this._sendingBackups = true;
+        try {
+            if (delay === undefined) {
+                // by default, wait between 0 and 10 seconds, to avoid backup
+                // requests from different clients hitting the server all at
+                // the same time when a new key is sent
+                delay = Math.random() * 10000;
+            }
+            if (delay > 0) {
```

Comment:
the `if` seems redundant here and just confuses things (even if it makes the 1 in 10,000 scenario that delay=0 infinitessimally faster)